src/patternfly/components/Slider/examples/Slider.md

```diff
@@ -0,0 +1,267 @@
+---
+id: 'Slider'
+beta: true
+section: components
+cssPrefix: pf-c-slider
+---
+
+## Examples
+### Discrete
+```hbs
+{{#> slider
+  slider--value-min="0"
+  slider--value-max="8"
+  slider--value-now="5"
+  slider--attribute='style="--pf-c-slider--value: 62.5%;"'
```

Comment:
whats this for

Reply:
It defines:

1. the track fill gradient stops to create the blue fill color - https://github.com/patternfly/patternfly/blob/a296e201f88e918a7ac0e335ec65c95b8b9978d2/src/patternfly/components/Slider/slider.scss#L9
1. positions the thumb - https://github.com/patternfly/patternfly/blob/a296e201f88e918a7ac0e335ec65c95b8b9978d2/src/patternfly/components/Slider/slider.scss#L36
1. positions the floating value input that displays over the thumb - https://github.com/patternfly/patternfly/blob/a296e201f88e918a7ac0e335ec65c95b8b9978d2/src/patternfly/components/Slider/slider.scss#L57

Basically all parts of the UI that update as the value of the slider changes so that the react component only has to update a single var and the slider CSS handles the layout.